### PR TITLE
#1597 init_DataLoading warnings for rawdata and population

### LINKED ISSUE #1597: When running Population only skip BIDS2legacy
### Description

bReadRawdata is true by default, unless the rawdata folder is missing. This is needed because by default we process the original scans in rawdata, not any derivatives. But in the case when a user does not want to process anything but just wants to do group analyses, running through BIDS2Legacy (even for checking that this was already done) may unnecessary slow things down.

In this case, the user can set bReadRawdata to false, which will skip BIDS2Legacy. We can automate this.

### Tasks

- [x] When rawdata exists, and the user gives [0 0 1] as processing booleans (== only running the population module), we give a warning that we will skip BIDS2Legacy and set bReadRawdata to false.
- [x] We give a warning if the user had bReadRawdata == true and processing [0 0 1], if the user for whatever reason wanted to both do BIDS2Legacy and running the population module only. 
----->>>>> the reason for this, is that the population module does a few things that DON'T NEED the structural & ASL modules. E.g., creating templates of the rawdata T1w, M0, or giving an overview of quantification parameters. But this probably won't happen a lot, which is why I propose this solution with a warning.

### Release notes

Don't run BIDS2Legacy conversion for re-running the population module.

### PR DESCRIPTION
**Pull request by jan-petr**
### Linked issue

Close #1597

### CODE REVIEW

File: Modules/Module_Processing/xASL_init_DataLoading.m [hunk 1]
```diff
@@ -63,6 +77,9 @@
         % Optionally, developers have set this parameter to force reading subjects from /rawdata
         warning(['rawdata folder missing: ' x.dir.RawData]);
         error('rawdata did not exist, if you want to read subjects from /derivatives/ExploreASL, set x.opts.bReadRawdata to false in dataPar.json');
+    elseif x.opts.bReadRawdata && isequal(x.opts.bProcess(:), [0; 0; 1])
```
Comment (HenkMutsaerts) [inline Modules/Module_Processing/xASL_init_DataLoading.m:78]: This is not true right?
If the user specifically set bReadRawdata to true, we don't change it

**Review by HenkMutsaerts — CHANGES_REQUESTED** [review]
Single suggestion, and I shortened the if-statement list a bit

File: Modules/Module_Processing/xASL_init_DataLoading.m (hunk 1 above)
Comment (jan-petr) [reply Modules/Module_Processing/xASL_init_DataLoading.m:78]: If use sets explicitly bReadRawdata to true, we don't change it. But I don't chante it :) I just issue a warning like agreed :)

All good? :)

Comment (HenkMutsaerts) [reply Modules/Module_Processing/xASL_init_DataLoading.m:78]: I think so. Did you check my last commit?

Comment (jan-petr) [reply Modules/Module_Processing/xASL_init_DataLoading.m:78]: Yes. It's all good and you can approve.

**Review by HenkMutsaerts — APPROVED** [review]

**Pull request merged**